File: src/main/java/com/learnershi/rclasssocket/entity/MiniWindow.java
package com.learnershi.rclasssocket.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.Accessors;
import org.springframework.data.mongodb.core.mapping.Document;

/**
 * Random
 * @author Sunhayng
 */
@Accessors(chain = true)
@NoArgsConstructor
@Getter
@Setter
@Document(collection = "miniWindow")
public class MiniWindow extends ClassRoomEntity {
    private Object newMiniWindow;  
}

File: src/main/java/com/learnershi/rclasssocket/entity/MiniWindow.kt
package com.learnershi.rclasssocket.entity

import org.springframework.data.mongodb.core.mapping.Document

/**
 * Random
 * @author Sunhayng
 */
@Document(collection = "miniWindow")
class MiniWindow : ClassRoomEntity() {
    private val newMiniWindow: Any? = null
}